perllib/FixMyStreet/App/Controller/Questionnaire.pm

```diff
@@ -206,16 +206,12 @@ sub submit_standard : Private {
         $update = $c->model('DB::Comment')->new(
             {
                 problem      => $problem,
-                name         => $problem->name,
```

Comment:
On the off chance they've used a different name subsequent to making the report removing this will change the behaviour. I'm not sure it's worth being concerned about though.

Reply:
Yeah, I spotted that, then I thought might be better anyway to use the name currently associated with the account filling it in at the time it's filled in 👍